auth/auth-server/src/server/api_auth.rs

```diff
@@ -40,7 +40,7 @@ impl Server {
 
         let key_description = self.check_api_key_auth(api_key, path, headers, body).await?;
         info!("Authorized request for entity: {key_description}");
-        Ok(())
+        Ok(key_description)
```

Comment:
nit: Let's add another line to the doc string for this method indicating that it returns a human readable description of the key